blitz_yang_telemetry/README.md

```diff
@@ -0,0 +1,32 @@
+Examples
+========
+
+This directory contains working examples run against Cisco devices.
+
+The testbed file has an XE Cat9K, and a Nexus 9k setup (Fake IP addresses).
+The mapping file allows you to filter out the device you want to test.
+The test_trigger_extended "data" file contains various xpath, namespace, and values that extends the test_trigger.
+The test_trigger file contains the actual definitions of the test and extends the data file.
+The job file is an example of how to choose exactly which test you wish to run.
+
+Main points to get your test running.
+
+- Testbed file device name must match the configured "host" name of the device.
+- TLS certificates will be needed if you are testing secure connections (see testbed file).
+- TLS certificate host override setting, if added to certificate, must be added to host file or DNS.
+- Mapping file must match the device name or alias you want to test in the testbed file.
+- Data file may need device name/connection of mapping file and the protocol used in test.
+- Job file, uncomment tests you want to run, comment out tests you don't want to run.
+
+Tests in test file:
+
+- device_cleanup - Not called in job file but there as an example.
```

Comment:
```suggestion
- device_cleanup - Not called in job file but there is an example.
```

Reply:
I *think* thats what it should be. Correct me if I'm wrong

Reply:
The device_cleanup test is literally in the test file and can be used "as" an example if the user fills in the values appropriate for their testbed.